pkg/porter/upgrade.go

```diff
@@ -71,7 +72,9 @@ func (p *Porter) UpgradeBundle(ctx context.Context, opts UpgradeOptions) error {
 		i.Bundle.Tag = ""
 	}
 
-	err = p.applyActionOptionsToInstallation(ctx, &i, opts.BundleActionOptions)
+	// FIXME: Should we check err here?
+	_ = p.applyActionOptionsToInstallation(ctx, &i, opts.BundleActionOptions)
```

Comment:
👀 This is not the only place where an error was returned and not checked. I was wondering if we needed to add some more checks. I think most of the time I didn't added some to avoid mutating the logic and failing in places where we didn't failed in the past. So here, I've just ignored it the value returned.

Reply:
We are adding in lint/vet checks that will catch these in the future and #2176 is fixing any problems in the current code base. So you don't need to worry about fixing that problem in this PR. 👍

Reply:
⚠️ **FIXME** (Note  to myself)